students/ganyoa/lessons/lesson01/assignment/inventory_management/main.py

```diff
@@ -0,0 +1,91 @@
+'''Launches the user interface for the inventory management system'''
+
+import sys
+import inventory_management.market_prices as market_prices
+import inventory_management.inventory_class as inventory_class
+import inventory_management.furniture_class as furniture_class
+import inventory_management.electric_appliances_class as electric_appliances_class
+
+'''
```

Comment:
Avoid living code commented out like this. If it's not required now, it probably won't be required latter. Also, triple quotes are actually strings and occupy memory while running code. If you need to comment code for some reason, use "#". On the other hand, docstrings are intended to be strings for documentation purposes (which is ok and it's how you're using them elsewhere in your code.